scripts/pre_commit_linter.py

```diff
@@ -468,7 +472,8 @@ def _redirect_stdout(new_target):
 
 
 def _lint_css_files(
-        node_path, stylelint_path, config_path, files_to_lint, stdout, result):
+        node_path, stylelint_path, config_path, files_to_lint,
+        stdout, result, mode):
```

Comment:
Hmm, this seems a bit weird, how about we go with a mid-way approach? Keep a `MODE` variable common for all functions (don't declare it with `global`) and set it according to the args parsed. Do a simple if-else inside all necessary functions. This looks cleaner than changing all functions. Thanks!

Reply:
Then we have to parse the args in every function where we want this mode.

Reply:
Nope, parse it once, store in a variable and share it among all functions, similar to [this variable](https://github.com/oppia/oppia/blob/develop/scripts/pre_commit_linter.py#L318).

Reply:
Okay thanks @apb7 !

Reply:
But there will be one issue in setting the value of the variable, since we are parsing the args in a function and to change a global variable we need to add a global statement.

Reply:
Hi @apb7, as you suggested to store in a variable like the other one you referred, so to change it global statement will be used.

Reply:
Need suggestions @apb7.

Reply:
Okay, I think we need to declare the mode as global only once when assigning it -- do this inside main() function. You do not need the keyword global when accessing its value.

Reply:
Done